src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier color;
+    private Random random;
+    private int bound = 100;
```

Comment:
should be also static and final. And do not forget that class constants should be uppercased